Configurations/unix-Makefile.tmpl

```diff
@@ -355,6 +355,15 @@ install_ssldirs:
 		cp $(SRCDIR)/apps/openssl.cnf $(DESTDIR)$(OPENSSLDIR)/openssl.cnf; \
 		chmod 644 $(DESTDIR)$(OPENSSLDIR)/openssl.cnf; \
 	fi
+	@echo "intsall $(SRCDIR)/util/ct_log_list.cnf -> $(DESTDIR)$(OPENSSLDIR)/ct_log_list.cnf.dist"
+	@cp $(SRCDIR)/util/ct_log_list.cnf $(DESTDIR)$(OPENSSLDIR)/ct_log_list.cnf.new
+	@chmod 644 $(DESTDIR)$(OPENSSLDIR)/ct_log_list.cnf.new
+	@mv -f  $(DESTDIR)$(OPENSSLDIR)/ct_log_list.cnf.new $(DESTDIR)$(OPENSSLDIR)/ct_log_list.cnf.dist
+	@if ! [ -f "$(DESTDIR)$(OPENSSLDIR)/ct_log_list.cnf" ]; then \
```

Comment:
Oh.  It was already used in openssl.cnf section just above.  Updated commit pushed; it fixes both.